frontend/src/admin/common/tables/ColumnHeader.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import styled from 'styled-components';
+import SortButton from './SortButton';
+
+const TH = styled.th`
+padding: 0;
+&:last-child {
+    width: 100px;
+}
+`;
+
+const ColumnHeader = ({ label, type = 'edit', onClick }) =>
+  <TH className={`${type}`} onClick={onClick}>
```

Comment:
Looks like the chevron is working. But I still don't quite understand what the className is for? It seems to result in DOM like this: 

<img width="321" alt="screen shot 2017-01-29 at 10 54 08 pm" src="https://cloud.githubusercontent.com/assets/1930451/22403777/f1d55738-e675-11e6-905d-59bc1b920c9a.png">

What are the name/tel/email/actions classes doing? This is on the admin or network admin tables btw.